src/main/java/com/google/sps/servlets/ProfileImage.java

```diff
@@ -51,7 +52,8 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       // Get the InputStream to store the file until it processed.
       InputStream fileInputStream = filePart.getInputStream();
 
-      UserImage.uploadObject("theglobetrotter-step-2020", fileName, fileInputStream);
+      Images.uploadObject(
+          "user-image-globes", "theglobetrotter-step-2020", fileName, fileInputStream);
```

Comment:
It would be better to make these strings constants somewhere, but not needed for this PR and could be done as future cleanup.